web/src/components/molecules/PluginsListCard/index.tsx

```diff
@@ -43,7 +43,7 @@ const PluginsListCard: React.FC<Props> = ({
       size="small"
       bordered={false}
       onClick={onClick}
-      bodyStyle={{ padding: "0" }}>
+      style={{ padding: "0" }}>
```

Comment:
Please follow the description /** @deprecated Please usestyles.body instead */

Reply:
done